packages/react-devtools-shared/src/devtools/views/Components/KeyValue.js

```diff
@@ -243,6 +243,13 @@ export default function KeyValue({
       displayValue = 'undefined';
     }
 
+    let shouldDisplayAsLink = false;
+    let protocolsAllowedAsLinks = ['file:///', 'http://', 'https://', 'vscode://'];
```

Comment:
nit: Move this array out of the render function. It doesn't need to be recreated each time the function renders.